ospf6d/ospf6_top.c

```diff
@@ -916,6 +987,7 @@ static void ospf6_show(struct vty *vty, struct ospf6 *o, json_object *json,
 		json_object_int_add(json, "holdTimeMultiplier",
 				    o->spf_hold_multiplier);
 
+		json_object_int_add(json, "maximumPaths", o->max_multipath);
```

Comment:
do you want to see this even if the value is the default? (same question about the vty version)

Reply:
This just tells us how many max-paths is supported in the display ipv6 ospf6. In the "show running configuration" I am not printing if its the default value